utils/scaling.py

```diff
@@ -39,8 +50,8 @@ def fscaling2D(X_train, X_test, scpatchSize, iscalefactor) :
                 BatchTest = icand
         dx_Train = None
         dx_Test = None
-        stepTrain = int(math.ceil(lenTrain/BatchTrain))
-        stepTest = int(math.ceil(lenTest/BatchTest))
+        stepTrain = -((0 - lenTrain) // BatchTrain)
```

Comment:
why not 
`lenTrain/ BatchTrain` ?

Reply:
Because the variable 'lenTrain' and 'BatchTrain' are integers, if I use 'int(math.ceil(lenTrain/BatchTrain))', the integer division will be done before the ceiling be executed.  
For example ,  int(math.ceil(6/5))=1 while int(math.ceil(6.0/5.0))=2. I changed the code so that the ceiling can take effect.

Reply:
alright, thanks for clarification